.ci/lint_r.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Do we really need two script files just to perform the R lint check? I guess we don't need to create a separate `.sh` file and litter the `.ci` folder to just launch another one script.

Reply:
oh sure! I thought it was a better separation to always have shell scripts doing orchestration. I don't mind changing this to just use the `.R` script and call it with `Rscript`. I'll do that.